src/permission/DefaultPermissions.php

```diff
@@ -106,6 +106,7 @@ public static function registerCorePermissions() : void{
 		self::registerPermission(new Permission(Names::COMMAND_UNBAN_IP, l10n::pocketmine_permission_command_unban_ip()), [$operatorRoot]);
 		self::registerPermission(new Permission(Names::COMMAND_UNBAN_PLAYER, l10n::pocketmine_permission_command_unban_player()), [$operatorRoot]);
 		self::registerPermission(new Permission(Names::COMMAND_VERSION, l10n::pocketmine_permission_command_version()), [$everyoneRoot]);
+		self::registerPermission(new Permission(Names::COMMAND_VERSION_PLUGINS, l10n::pocketmine_permission_command_version_plugins()), [$operatorRoot]);
```

Comment:
This will cause problems for people who set `pocketmine.command.version` expecting it to set this permission as well.

It's probably better to have:
- `pocketmine.command.version.server`
- `pocketmine.command.version.plugins`

and then have `pocketmine.command.version` as a deprecated backwards-compatible group permission for the remainder of PM5. See how this was done for `/effect` in PM4: https://github.com/pmmp/PocketMine-MP/blob/ba89ae5bf20dcc4aa4d40698429535aad26cc79d/src/permission/DefaultPermissions.php#L71-L73

This will permit people to still explicitly set `pocketmine.command.version` to get the same functionality as it would previously, but since it won't be given to any groups, default functionality will be controlled by these new permissions.

(Side note for maintainers: If #6324 were implemented, explicitly registering deprecated group roots wouldn't be necessary anymore.)